testing/src/cypress/e2e/02-institution/06-manage-users.cy.ts

```diff
@@ -86,7 +86,7 @@ describe("Manage Users - Add new user", () => {
       .should("have.text", "Admin");
     manageUsersObject
       .isLegalSigningAuthorityButton()
-      .should("have.text", "Legal signing authority");
+      .should("have.text", "Legal Signing Authority");
```

Comment:
Is it case sensitive?

Reply:
Yeah the string validation is case sensitive.